### DISCUSSION TITLE
Adds verification flag for third-party buckets, otherwise they will not be found by default when searching for packages. You must uncheck official buckets only.

### DISCUSSION THREAD

**Discussion by kjxbyz** [ScoopInstaller/Scoop#6020] [Ideas]
Adds verification flag for third-party buckets, otherwise they will not be found by default when searching for packages. You must uncheck official buckets only.

**Comment by HUMORCE** [comment]
https://github.com/ScoopInstaller/scoopinstaller.github.io/issues/87 Create based on your demand.
